bundles/io/org.eclipse.smarthome.io.console/src/main/java/org/eclipse/smarthome/io/console/extensions/GenericConsoleCommandExtension.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.console.extensions;
+
+import org.eclipse.smarthome.io.console.Console;
+
+/**
+ * A base class that should be used by console command extension for better inclusion.
+ *
+ * @author Markus Rathgeb - Initial contribution and API
+ *
+ */
+public abstract class GenericConsoleCommandExtension implements ConsoleCommandExtension {
```

Comment:
Thanks, will change that.
Generic is for me something that is not specific (so nothing that is concrete IMHO), but Abstract or Base is better, you're right.